credits/zh-cn.json

```diff
@@ -0,0 +1,16 @@
+{
+    "developer": "开发",
+    "designer": "设计",
+    "project_manager": "项目管理",
+    "qa_engineer": "质量保障",
+    "scenarist": "脚本",
+    "achievement_manager": "成就管理",
+    "producer": "制作人",
+    "joke_teller": "笑话",
```

Comment:
I cannot confirm from translations, is this the right term for "the person who tells/writes the jokes" ?

Reply:
"笑话" is the word "Joke". Only in rep of myself, it is not usual to see "讲笑话的人/Joke-teller" in Chinese, but it can be okay to use a noun in credit list to mean "the provider of something".

Reply:
I wasn't sure this was "Joke" 😄 But it's okay with that translation then. Maybe if there's a difference for a plural version, it would be better (but I don't know how it work in Chinese so it may make no sense ahah)

Reply:
Grammatically we don't change the word to mean plural in Chinese haha. Using the word directly should indicate the amount is not important in the syntax -- or just to represent a genre of things. 

Therefore I think "笑话" is good enough.

Reply:
Okay, then we keep this :)